layer1/monitor/monitor.go

```diff
@@ -389,14 +389,16 @@ func PersistSnapshot(eth layer1.Client, bh *objs.BlockHeader, numOfValidators in
 	}
 
 	// kill any snapshot task that might be running
-	ctx := context.Background()
-	_, err = taskHandler.KillTaskByType(ctx, &snapshots.SnapshotTask{})
+	_, err = taskHandler.KillTaskByType(&snapshots.SnapshotTask{})
 	if err != nil {
 		return err
 	}
 
-	_, err = taskHandler.ScheduleTask(ctx, snapshots.NewSnapshotTask(uint64(bh.BClaims.Height), numOfValidators, validatorIndex), "")
-	return err
+	_, err = taskHandler.ScheduleTask(snapshots.NewSnapshotTask(uint64(bh.BClaims.Height), numOfValidators, validatorIndex), "")
+	if err != nil {
+		return err
+	}
+	return nil
```

Comment:
Should just leave this as `return err`, the if statement doesn't do anything.

Reply:
I remember Hunter suggesting me the opposite in a similar situation. The motivation was if I just do `return err` here, potentially in the future someone can add new code after and forget to add that if. So it's just for kinda safety.